libres/lib/enkf/analysis_config.cpp

```diff
@@ -146,8 +146,8 @@ void analysis_config_set_max_runtime(analysis_config_type *config,
     config->max_runtime = max_runtime;
 }
 
-static void analysis_config_set_min_realisations(analysis_config_type *config,
-                                                 int min_realisations) {
+void analysis_config_set_min_realisations(analysis_config_type *config,
```

Comment:
Seems to me like this function is only used in `test_assert_update_viable.cpp` and in `analysis_config.cpp`.
If we are not to use it anywhere else, should we declare it in `test_assert_update_viable.cpp` instead of declaring it in a header?
This is what we've done with other "static" functions.

Reply:
I agree with this idea